independent-projects/bootstrap/app-model/src/main/java/io/quarkus/bootstrap/util/PropertyUtils.java

```diff
@@ -67,4 +77,166 @@ public static final boolean getBoolean(String name, boolean notFoundValue) {
         final String value = getProperty(name, (notFoundValue ? TRUE : FALSE));
         return value.isEmpty() ? true : Boolean.parseBoolean(value);
     }
+
+    /**
+     * Stores properties into a file sorting the keys alphabetically and following
+     * {@link Properties#store(Writer, String)} format but skipping the timestamp and comments.
```

Comment:
How exactly do you skip the timestamps?

Reply:
By writing each line in the methods below instead of calling `Properties.store(...)`

Reply:
Ah OK. It's `Properties.store()` adding the timestamps!

Reply:
You can bypass that by running with `-Djava.properties.date=`, but I think the current solution is easier to understand